lib/tdf3/src/models/attribute-set.ts

```diff
@@ -1,4 +1,4 @@
-import { Validator } from 'jsonschema';
+import Ajv from 'ajv';
```

Comment:
I like `ajv`, but what was the reasoning to switch?

Reply:
Test running in the browser showed that it had node js dependency that was not recognized by browser. So ajv is browser first lib